### PR TITLE
[luci/pass] Implement FusePReluPass

### PR DESCRIPTION
**Pull request by seanshpark**
This will implement FusePReluPass to fuse certain subgraph to CirclePRelu Op.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #10359 from #10361

File: compiler/luci/pass/src/FusePReluPass.cpp [hunk 1]
```diff
@@ -15,17 +15,173 @@
...
+  luci::CircleAbs *_abs = nullptr;
+  luci::CircleSub *_sub = nullptr;
+  luci::CircleMul *_mul_alpha = nullptr;
+  luci::CircleMul *_mul_half = nullptr;
+  luci::CircleAdd *_add_ofm = nullptr;
+  luci::CircleConst *_const_alpha = nullptr;
+  luci::CircleConst *_const_half = nullptr;
+
+private:
+  bool _matched = false;
```
Comment (jinevening) [inline compiler/luci/pass/src/FusePReluPass.cpp:85]: Is this variable necessary?

File: compiler/luci/pass/src/FusePReluPass.cpp [hunk 2]
```diff
@@ -15,17 +15,173 @@
...
+};
+
+#define CHECK_OR_FALSE(condition) \
+  if (not(condition))             \
+    return false;
+
+bool PReluPattern::match()
```
Comment (jinevening) [inline compiler/luci/pass/src/FusePReluPass.cpp:92]: CircleMul, CircleSub, CircleAdd can have fused act function.

They have to be None.

File: compiler/luci/pass/src/FusePReluPass.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/pass/src/FusePReluPass.cpp:85]: This variable came from InstNormPass :)
I'll remove this.

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**